qpc/insights/tests_insights_configure.py

```diff
@@ -0,0 +1,87 @@
+"""Test the CLI module."""
+
+import sys
+import unittest
+from io import StringIO
+
+from qpc.cli import CLI
+from qpc.tests_utilities import HushUpStderr, redirect_stdout
+from qpc.utils import read_insights_config, write_insights_config, write_server_config
+
+DEFAULT_PORT = 443
+DEFAULT_HOST = 'console.redhat.com'
+
+DEFAULT_CONFIG = {'host': '127.0.0.1', 'port': 8000, 'use_http': True, 'require_token': False}
+
+
+class InsightsConfigureTests(unittest.TestCase):
```

Comment:
I'd mock `qpc.utils.QPC_INSIGHTS_CONFIG` to a [tempfile](https://docs.python.org/3/library/tempfile.html)

This will:
1. Protect our personal config file (if we have one)
2. Protect this and other tests that might interfere with this config (even though we'd probably be better mock `read_insights_config`)

Reply:
This is now solved on the commit "Adds INSIGHTS_CONFIG file to mock_path_constants" (current sha: 4d4f75df549cea1ad4f3dc81cd7d27e87f2f1d01)